API_Guidelines.md

```diff
@@ -97,6 +97,31 @@ Payload
 }
 ```
 
+## POST
+
+### Save Method
+
```

Comment:
adopt format already in GET. 'Request' heading with URL and Ember Data Method <dl>